Update Read me File (#19)
medi-book/medibook-ui: README.md

![](https://img.shields.io/badge/Microverse-blueviolet)

# Book an appointment Front-end

> The Project we are going to build is for Final Capstone Project. Doctor Appointment frontend is react frontend application for scheduling doctor appointments.Users need to sign up or sign in to access application.
> 

Reviewer: ```suggestion
This project is our Final Capstone Project. It's a Doctor appointment scheduling app. Built with a React frontend and Rails backend. Users need to sign up or sign in to access the application.

```

Reviewer: Also, let's add a link to the backend in the readme

Author: Yes I already Added
